src/NuGet.Services.Configuration/Properties/AssemblyInfo.cs

```diff
@@ -7,6 +7,6 @@
 [assembly: AssemblyCompany(".NET Foundation")]
 [assembly: AssemblyCopyright("Copyright © .NET Foundation 2016")]
 [assembly: ComVisible(false)]
-[assembly: AssemblyVersion("2.0.0")]
-[assembly: AssemblyFileVersion("2.0.0")]
-[assembly: AssemblyInformationalVersion("2.0.0-rc2-1")]
```

Comment:
>nalVersion("2.0.0-rc2-1")] [](start = 29, length = 26)

this shouldn't be changed, since it's the version of the NuGet.Services.Configuration assembly.